packages/react/popover/src/Popover.tsx

```diff
@@ -260,6 +263,7 @@ const PopoverContentImpl = React.forwardRef((props, forwardedRef) => {
           // we make sure we're not trapping once it's been closed
           // (closed !== unmounted when animating out)
           trapped={trapFocus && context.open}
+          unmountAutoFocusElement={context.trigger}
```

Comment:
Are we going to be explicit in all the use-cases of `FocusScope`? If so, should it be a required prop and not bother about `document.activeElement` as a fallback?

I guess there are still places like context menu where it's the default.
Makes me think though, I think it was in `DropdownMenu` that we forced focusing the trigger, because we used `mousedown` to trigger open, which meant that activeElement wasn't set on time, so that seems to be a good place for your new prop too, rather than preventing default in the `onCloseAutoFocus` and running our own focus.

Reply:
> Are we going to be explicit in all the use-cases of `FocusScope`? If so, should it be a required prop and not bother about `document.activeElement` as a fallback?

I thought about this too but we use `FocusScope` in `Menu` which has no trigger so couldn't really make it required there. Same for `ContextMenu`, if this was required what would I pass there?